styles.css

```diff
@@ -113,7 +113,7 @@ html {
 }
 
 .paragraph-style label:hover {
-  background-color: #f15f31;
+  background-color: #0d7ebf;
```

Comment:
Try using css-variables

Reply:
I used CSS variables with the main colors of the page. Thank you.